General/general_shacl.ttl

```diff
@@ -0,0 +1,206 @@
+
+general:LicenceShape a sh:NodeShape ;
+    sh:property [ skos:example "link to licence" ;
+            sh:datatype xsd:anyURI ;
+            sh:description "Reference to licence"@en ;
+            sh:maxCount "1"^^xsd:unsignedInt ;
+            sh:message "Validation of link failed!"@en ;
+            sh:name "link"@en ;
+            sh:order "1"^^xsd:unsignedInt ;
+            sh:path general:link ],
+        [ skos:example "MIT" ;
+            sh:datatype xsd:string ;
+            sh:description "Name of licence type"@en ;
+            sh:maxCount "1"^^xsd:unsignedInt ;
+            sh:message "Validation of type failed!"@en ;
+            sh:name "type"@en ;
+            sh:order "0"^^xsd:unsignedInt ;
+            sh:path general:type ] ;
+    sh:targetClass general:Licence .
+
+general:LinksShape a sh:NodeShape ;
+    sh:property [ skos:example "link to CRG" ;
+            sh:description "Reference to dependent data"@en ;
+            sh:message "Validation of data failed!"@en ;
+            sh:minCount "0"^^xsd:unsignedInt ;
+            sh:name "data"@en ;
+            sh:node general:LinkShape ;
+            sh:order "1"^^xsd:unsignedInt ;
+            sh:path general:data ],
+        [ skos:example "link to screenshot, video, routing" ;
+            sh:description "Reference to screenshot data"@en ;
+            sh:message "Validation of media failed!"@en ;
+            sh:name "media"@en ;
+            sh:node general:LinkShape ;
+            sh:order "2"^^xsd:unsignedInt ;
+            sh:path general:media ],
+        [ skos:example "link to asset data, e.g. hd map" ;
+            sh:datatype xsd:anyURI ;
+            sh:description "Id / url to asset file"@en ;
+            sh:maxCount "1"^^xsd:unsignedInt ;
+            sh:message "Validation of asset failed!"@en ;
+            sh:name "asset"@en ;
+            sh:order "0"^^xsd:unsignedInt ;
+            sh:path general:asset ] ;
+    sh:targetClass general:Links .
+
+general:PositionShape a sh:NodeShape ;
+    sh:property [ skos:example "Lat: 1.234- 1.345; Lon 45.23-45.34" ;
+            sh:description "Bounding box with lat/lon values in WGS84"@en ;
+            sh:maxCount "1"^^xsd:unsignedInt ;
+            sh:message "Validation of bounding failed!"@en ;
+            sh:name "bounding"@en ;
+            sh:node general:BoundingBoxShape ;
+            sh:order "1"^^xsd:unsignedInt ;
+            sh:path general:bounding ],
+        [ skos:example "Germany, GER" ;
+            sh:datatype xsd:string ;
+            sh:description "Country code as ISO 3166-1, alpha-2; "@en ;
+            sh:maxCount "1"^^xsd:unsignedInt ;
+            sh:message "Validation of country failed!"@en ;
+            sh:name "country"@en ;
+            sh:order "0"^^xsd:unsignedInt ;
+            sh:path general:country ;
+            sh:pattern "^[a-zA-Z]{2}-(?:[a-zA-Z]{1,3}|[0-9]{1,3})$" ] ;
+    sh:targetClass general:Position .
+
+general:PriceShape a sh:NodeShape ;
```

Comment:
do we need a shape for a single attribute?
If price should be used somewhere else also then this is ok.

Reply:
as discussed, the price is removed because it is shown in the marketplace itself and is not part of the metadata

Reply:
PriceShape is still existing in the file.

Reply:
price is gone, resolved